docs/platforms/javascript/common/enriching-events/scopes/index.mdx

```diff
@@ -8,35 +8,97 @@ information from the current scope. SDKs will typically automatically manage the
 for you in the framework integrations and you don't need to think about them. However,
 you should know what a scope is and how you can use it for your advantage.
 
-## What's a Scope, What's a Hub
-
-You can think of the hub as the central point that our SDKs use to route an
-event to Sentry. When you call `init()` a hub is created and a client and a
-blank scope are created on it. That hub is then associated with the current
-thread and will internally hold a stack of scopes.
+## What's a Scope?
 
 The scope will hold useful information that should be sent along with the
 event. For instance [contexts](../context/) or
 [breadcrumbs](../breadcrumbs/) are stored on
-the scope. When a scope is pushed, it inherits all data from the parent scope
-and when it pops all modifications are reverted.
+the scope. When a scope is forked, it inherits all data from the parent scope.
+
+The default SDK integrations will fork scopes intelligently. For
+instance web framework integrations will fork scopes around your
+routes or request handlers.
+
+## How Scopes Work
+
+Scopes are basically a stack of data that is attached to events. When an event is captured, the SDK will merge the data from the current scope into the event. This allows you to attach data to events that is relevant to the context in which the event was captured.
```

Comment:
```suggestion
Scopes are basically a stack of data that is attached to events. When an event is captured, the SDK will merge the data from the active scopes into the event. This allows you to attach data to events that is relevant to the context in which the event was captured.
```
Hope I'm not misunderstanding, but it sounds like data from all active scopes (global, isolation, current) is applied to the event, not just the current scope.